megalinter/descriptors/lua.megalinter-descriptor.yml

```diff
@@ -22,12 +30,7 @@ linters:
         - readline-dev
```

Comment:
Ok, so after the tests of building individual linter images in my fork, I found out that this readline-dev is a dependency of lua itself, but openssl is only required for a library used by luacheck, which uses either that, md5sum or md5. 

So, this needs to be moved to the language-level install, but openssl kept here.